tutorials/dataframe/df025_RNode.C

```diff
@@ -1,38 +1,43 @@
 /// \file
 /// \ingroup tutorial_dataframe
 /// \notebook
-/// RNode is a generic type which represents any transformation node in the computation graph.
-/// This tutorial shows how to take advantage of the RNode class.
+/// Each RDataFrame object has its own type. It helps with performance,
```

Comment:
Add a brief docstring that doxygen can show [here](https://root.cern/doc/master/group__tutorial__dataframe.html).
```suggestion
/// Manipulating computation graphs with RNode.
///
/// Each RDataFrame object has its own type. It helps with performance,
```